File: 0333/333.go
func largestBstSubtree(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := largestBstSubtree(root.Left)
	r := largestBstSubtree(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 0333/333.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var largestBstSubtree = function(root) {
    if (!root) return 0;
    return 1 + Math.max(largestBstSubtree(root.left), largestBstSubtree(root.right));
};